### PR TITLE
Travis runs on Bionic 18.04

### PR DESCRIPTION
**Pull request by ca-johnson**
Default on travis is Xenial 16.04
Bionic 18.04 is more widely used at Improbable

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ python:
   - "3.6"
 os:
   - linux
+dist: bionic # 18.04
```
Comment (petemounce) [inline .travis.yml:7]:
```suggestion
dist: 
  - bionic # 18.04
  - focal  # 20.04 LTS
```

I _think_ this is possible?

**Review by petemounce — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (ca-johnson) [reply .travis.yml:7]: Tried it and it just ran on bionic :/ https://travis-ci.com/github/improbable-eng/perforce-buildkite-plugin/builds/182578913

I think there is a way, may need to write something like:
```
jobs:
  - os: linux
    dist: bionic
  - os:linux
    dist: focal
```

Just gonna leave it for now - mainly just wanted to run on the same version we are actually using in prod

Comment (petemounce) [reply .travis.yml:7]: Ah, ok.

We're using 16.04 LTS in prod currently.

Comment (ca-johnson) [reply .travis.yml:7]: Oh so we are.. oops! Not sure why I had 18.04 in my head..

Comment (petemounce) [reply .travis.yml:7]: Rampant unbridled lust, I expect ;)

**Pull request closed without merging**